### ISSUE TITLE
deprecated prometheus log package breaks build

### ISSUE THREAD

**Issue by aqw** [czerwonk/ping_exporter#60]
I am packaging this exporter, and current the build fails with:
```
...
   dh_auto_build -O--builddirectory=_build -O--buildsystem=golang
        cd _build && go install -trimpath -v -p 8 github.com/czerwonk/ping_exporter github.com/czerwonk/ping_exporter/config
src/github.com/czerwonk/ping_exporter/target.go:11:2: cannot find package "github.com/prometheus/common/log" in any of:
        /usr/lib/go-1.17/src/github.com/prometheus/common/log (from $GOROOT)
...
```
87ca369e6b2147519ea36667514d3e8856fc2216 replaced `prometheus/common/log` with `sirupsen/logrus`, but it seems that `target.go` (line 11) was missed.

**Comment by soapiestwaffles** [comment]
@aqw this is fixed now in the latest `master` after #64 was merged!

**Comment by aqw** [comment]
Thank you! Closing.
